### DISCUSSION TITLE
Is it possible to make rego throw an error (or exception)?

### DISCUSSION THREAD

**Discussion by renatosc** [open-policy-agent/community#474] [OPA and Rego]
I want to enforce that a timestamp that I receive as input to be of UTC timezone and if it is not, I would like to OPA to stop evaluating the rule (so, no response) and log that as an error. Is it possible to do that today? I looked on the docs and I didn't find a way to do that.

Thank you

**Comment by anderseknert** [comment]
Did you see https://www.openpolicyagent.org/docs/latest/policy-language/#errors ? I'm on my phone (vacation, yay!) so I've not been able to try the query parameter mentioned there, but maybe that's what you're looking for. The docs aren't clear on if/what/how it changes the response / logging though. That'd be a good improvement, I think.

**Reply by renatosc** [reply]
Thanks for the reply @anderseknert. I had searched the docs and I don't know how did I miss that.

I will try using that `strict-builtin-errors` flag.
